avm/src/vm/opcode.rs

```diff
@@ -2982,16 +2988,107 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
   });
 
   async fn http_listener(req: Request<Body>) -> VMResult<Response<Body>> {
+    // Grab the headers
+    let headers = req.headers();
+    // Check if we should load balance this request
+    if !headers.contains_key("x-alan-rr") {
```

Comment:
can we expand the comment a bit to explain how the load balancing works with this secret header?

Reply:
Sure